### PR TITLE
all-in-one template

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

**Comment by pavolloffay** [comment]
@jpkrohling @objectiser could you please review? 

We should maybe establish a file structure conventions. E.g. create a separate folder for `all-in-one`. 

I will need a tag in this repo for MSA example.

**Comment by objectiser** [comment]
@pavolloffay agree we should have some structure to separate - could be all-in-one or standalone as used in jaeger repo.

**Comment by pavolloffay** [comment]
ok so I will move it to `all-in-one` folder

**Comment by objectiser** [comment]
Do we want to provide more comprehensive usage instructions, as in https://github.com/jboss-dockerfiles/hawkular-apm/tree/master/openshift-templates ?

i.e. for people who are new to openshift, and using this to experiment with openshift + jaeger - unless there is somewhere else we can point them to? @jpkrohling thoughts?

**Comment by pavolloffay** [comment]
Good point I will add more instructions, one liner which would run it all.

**Comment by jpkrohling** [comment]
This is what I have in my WIP branch: https://github.com/jpkrohling/jaeger/blob/aa5a0a273aa818e9eecd26a6d1e02beb826dbe34/examples/openshift/README.md , in case you want to use that as base for this one.

**Comment by pavolloffay** [comment]
I don't think that somebody unfamiliar with OC would use these templates. Maybe we can point interested users to some tutorial or OC documentation but I don't think that providing a starting guide to OC is a subject of this repo.

**Comment by jpkrohling** [comment]
Indeed, but people trying to deploy their own applications to OpenShift might need some help in understanding what's going on and what exactly to execute, in order to get a Jaeger instance on OpenShift. On the README I linked before, I have only a small section pointing users to the installation guide for OpenShift, everything else is related to the template itself.

**Comment by objectiser** [comment]
For now we could just have simple instructions with the link mentioned by Juca, and then review overtime depending on the type of users trying it out.

File: README.md [hunk 1]
```diff
@@ -11,3 +16,10 @@ To directly install everything you need:
 oc process -f https://raw.githubusercontent.com/jaegertacing/jaeger-openshift/master/all-in-one/jaeger-all-in-one-template.yml | oc create -f -
 oc delete all,template -l jaeger-infra    # to remove everything
 ```
+
+## Troubleshooting
+Tracer sometimes fails to resolve Jaeger's address. In such a case run:
+```bash
+sudo iptables -F
+```
+and restart applications affected applications.
```
Comment (objectiser) [inline README.md]: Remove first `applications`

Comment (pavolloffay) [inline README.md]: applications^2

Comment (pavolloffay) [inline README.md]: :+1: removed

**Comment by pavolloffay** [comment]
@objectiser @jpkrohling thanks!

**Pull request merged**